1.Exploratory_Data_Analysis/features/PBMC_feature_index.txt

```diff
@@ -50,16 +48,15 @@ Cytoplasm_Correlation_Overlap_CorrDNA_CorrPM
 Cytoplasm_Correlation_Overlap_CorrGasdermin_CorrMito
 Cytoplasm_Correlation_Overlap_CorrGasdermin_CorrPM
 Cytoplasm_Correlation_Overlap_CorrMito_CorrPM
-Cytoplasm_Correlation_RWC_CorrDNA_CorrER
-Cytoplasm_Correlation_RWC_CorrDNA_CorrMito
-Cytoplasm_Correlation_RWC_CorrDNA_CorrPM
+Cytoplasm_Correlation_RWC_CorrER_CorrDNA
```

Comment:
Do you know why these are different? Is it b/c feature selection is different? (just curious)

Reply:
I believe so, yes.